content/blog/getting-started-with-spark-on-mapr-sandbox.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting Started with Spark on MapR Sandbox
+date: 2021-10-14T06:01:00.000Z
+author: MapR Tutorials
+authorimage: /img/Avatar1.svg
+---
+
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/ezmeral-data-fabric.html>
+
+At MapR, we distribute and support Apache Spark as part of the MapR Converged Data Platform. This tutorial will help you get started with running Spark applications on the MapR Sandbox.
+
+## Prerequisites
+
+**HARDWARE REQUIREMENTS**
+
+* 8GB RAM, multi-core CPU
+* 20GB minimum HDD space
+* Internet access
+
+**SOFTWARE REQUIREMENTS**
+
+* A hypervisor. This example uses VMware Fusion 6.0.2 on OSX; however, other VMware products could be used instead. Additionally, [VirtualBox](https://www.virtualbox.org/) can be used
+* A virtual machine image for the MapR Sandbox. Spark comes pre-loaded with version 5.0 and later of the MapR Sandbox.
+
+**Starting up and Logging into the Sandbox**\
```

Comment:
Please delete the \ at the end of the header "Starting up and Logging into the Sandbox"

Reply:
Agree with Denis' comment. Currently, the blog post shows in the preview as being oddly formatted at this point due to this. Also, please change "login" to "log in" for correct grammar.
![image](https://user-images.githubusercontent.com/77496261/137519874-ed989b66-35bd-4740-911b-b3e6a9309313.png)

Reply:
done